.devcontainer/on-create.sh

```diff
@@ -2,18 +2,33 @@
 
 # this runs as part of pre-build
 
-echo "$(date)    on-create start" >> $HOME/status
+echo "on-create start"
+echo "$(date +'%Y-%m-%d %H:%M:%S')    on-create start" >> "$HOME/status"
 
 # clone repos
-git clone https://github.com/retaildevcrews/ngsa-app /workspaces/ngsa-app
+git clone https://github.com/cse-labs/imdb-app /workspaces/imdb-app
 git clone https://github.com/microsoft/webvalidate /workspaces/webvalidate
 
+export REPO_BASE=$PWD
+export PATH="$PATH:$REPO_BASE/bin"
+
+mkdir -p "$HOME/.ssh"
+mkdir -p "$HOME/.oh-my-zsh/completions"
+
+{
+    # add cli to path
+    echo "export PATH=\$PATH:$REPO_BASE/bin"
+
+    echo "export REPO_BASE=$REPO_BASE"
+    echo "compinit"
+} >> "$HOME/.zshrc"
+
 # restore the repos
 dotnet restore /workspaces/webvalidate/src/webvalidate.sln
-dotnet restore /workspaces/ngsa-app/Ngsa.App.csproj
+dotnet restore /workspaces/imdb-app/src/imdb.csproj
 
 # copy grafana.db to /grafana
-sudo cp deploy/grafanadata/grafana.db /grafana
+sudo cp .devcontainer/grafana.db /grafana
```

Comment:
grafana.db is going away (please see other PR)

Reply:
This should go away with sequential pr #49